test/safe/push.test.ts

```diff
@@ -8,12 +8,17 @@ import { Application } from "probot";
 import { start, stop } from "../../src/worker/startup";
 import { booleanFlag, BooleanFlags } from "../../src/config/feature-flags";
 import { when } from "jest-when";
+import waitUntil from "../utils/waitUntil";
+import sqsQueues from "../../src/sqs/queues";
 
 jest.mock("../../src/config/feature-flags");
 
 const createJob = (payload, jiraHost: string) => ({ data: createJobData(payload, jiraHost) } as any);
 
 describe("Push Webhook", () => {
+
+	const realDateNow = Date.now.bind(global.Date);
```

Comment:
Could you use `fakeTimers` from jest?

Reply:
Discussed F2F, will try fakeTimers and remove this code when we get rid of Redis